app/components/market/submit/submit-market-application.html

```diff
@@ -1,37 +1,223 @@
 <bl-scrollable>
-    <bl-complex-form [formGroup]="form" [submit]="submit" [hideFooter]="true" *ngIf="form">
-        <bl-form-page main-form-page [title]="title" [formGroup]="form">
-            <bl-form-section title="Job" subtitle="Job parameters" formGroupName="job">
-                <md-input-container class="form-element" *ngIf="jobNameParam">
-                    <input mdInput type="text" name="name" formControlName="jobName" placeholder="Job name">
-                </md-input-container>
-                <div *ngFor="let param of otherParameters" [ngSwitch]="param.type">
-                    <div *ngSwitchCase="NcjParameterExtendedType.fileGroup">
-                        <bl-file-group-picker [formControlName]="param.id" [label]="param.name" class="form-element" [hint]="param.description">
-                        </bl-file-group-picker>
+    <div *ngIf="mode_state == 'None'">
+        <bl-complex-form [formGroup]="form" [hideFooter]="true">
+            <bl-form-page main-form-page [title]="title">
+                <bl-form-section title="Mode">
+                    <ul>
+                        <button style="margin:5px; width:200px" (click) = "mode_state = 'PoolNJob'" type="button" md-raised-button>new Pool + Job</button>
+                        <button style="margin:5px; width:200px" (click) = "mode_state = 'PoolOJob'" type="button" md-raised-button>existing Pool + Job</button>
+                        <button style="margin:5px; width:200px" (click) = "mode_state = 'Pool'" type="button" md-raised-button>new Pool</button>
+                    </ul>
+                </bl-form-section>
+            </bl-form-page>
+        </bl-complex-form>
+    </div>
+
+    <div *ngIf="mode_state == 'PoolNJob'">
```

Comment:
There is also an inconsistent use of ' and ".

Is the linter not catching these?